### PR TITLE
Allows admins to make a dance machine inmovable, makes a dance machine subtype that's immovable and unbreakable, and makes them have 400 health.

### PR DESCRIPTION
**Pull request by silicons**
Kek.

### CODE REVIEW

**Comment by Metallusions** [comment]
Do dance machines affect nearby ghosts?

**Comment by PKPenguin321** [comment]
isnt the anchored var a thing already

**Comment by PKPenguin321** [comment]
oh wait this has special checks
if you do this make a special path that is also indestructible please

**Comment by Bawhoppen** [comment]
admincreep

File: code/game/machinery/dance_machine.dm [hunk 1]
```diff
@@ -25,6 +28,10 @@
 		)
 	var/datum/track/selection = null
 
+/obj/machinery/disco/immovable
```
Comment (Cyberboss) [inline code/game/machinery/dance_machine.dm]: Name it something else, since this also give it `INDESTRUCTIBLE`

**Review by Cyberboss — CHANGES_REQUESTED** [review]

**Comment by Robustin** [comment]
On one hand, whatever. On the other hand. This IS going to be an engineering reward and part of its value is in the unique experience you get from earning it. Encouraging admins to poop one out whenever they want isn't really compatible with that.

**Comment by Cyberboss** [comment]
Admins can already poop one out if they want, so the argument is kinda
invalid.

**Comment by Robustin** [comment]
Yea but it reinforces the idea that its ok for admins to drop one whenever they want.

**Comment by Cyberboss** [comment]
If it exists in game, admins will drop it whenever they want.

**Comment by kevinz000** [comment]
@Robustin not gonna wait for the PR that will never go through and admins can do it anyways

**Comment by Robustin** [comment]
That PR has the triumvirate's support - again this isn't a big deal but I hope these machines show up sparingly. Much of their novelty is in their rarity, I can testify first-hand that I loved this machine to death in the first hours of testing but after hearing the same song & dance for the 100th time it lost all effect on me.

**Comment by optimumtact** [comment]
what is the triumvirate

**Comment by KorPhaeron** [comment]
I agree with robustin that admins spawning PvE rewards and spamming new features cheapens them. No reason to do your job if you are just given it.

**Comment by kevinz000** [comment]
@KorPhaeron Yeah but right now there's no engi points PR, so this is the only way to let players use it. 
@Robustin on the bright side because people like it when I used it I'm sure they'll be willing to use your feature when you actually make it work :^)

**Comment by Robustin** [comment]
It just reeks a little of opportunism too when you try to get my PR's killed, then loot the remains, then come back and say "actually I want to make spawning this more convenient for myself" and use the excuse that I don't have the PR's open to justify it. I'm ok with the machine showing up from time to time (I'd actually like to see it tested again after my fixes to see if there's anything I missed) but an admin-version would send the message that this is an admin toy and even if a system isn't in place right now, it would lose much of its appeal if it got spawned to death before players could even earn it naturally. While I appreciate your improvements to the machine, I know that attention comes from you wanting to spawn this device more regularly - hence my concern. 

The Engi system is next on the list after I finish this cult stuff.

**Comment by PKPenguin321** [comment]
?

**Comment by optimumtact** [comment]
Kor said no

**Pull request closed without merging**